networkx/algorithms/shortest_paths/generic.py

```diff
@@ -98,17 +98,17 @@ def shortest_path(G, source=None, target=None, weight=None, method="dijkstra"):
     Examples
     --------
     >>> G = nx.path_graph(5)
-    >>> print(nx.shortest_path(G, source=0, target=4))
+    >>> print(nx.shortest_path(G, source=0, target=4)) # shortest path from source=0 to source=4
```

Comment:
```suggestion
    >>> print(nx.shortest_path(G, source=0, target=4))
```

This is probably not required :)